src/goto-checker/symex_bmc.cpp

```diff
@@ -212,5 +214,12 @@ void symex_bmct::no_body(const irep_idt &identifier)
   {
     log.warning() << "**** WARNING: no body for function " << identifier
                   << log.eom;
```

Comment:
Did you mean to delete this one because this will give a duplicate `log.eom` in the normal case.

Reply:
deleted